lang/java/reef-common/src/main/java/org/apache/reef/driver/restart/DriverRestartManager.java

```diff
@@ -40,16 +38,14 @@
 public final class DriverRestartManager {
```

Comment:
Other than isRestartCompleted, the rest of the methods will be used in future commits.